test/built-ins/RegExp/match-indices/indices-groups-properties.js

```diff
@@ -7,22 +7,22 @@ includes: [compareArray.js, propertyHelper.js]
 esid: sec-makeindicesarray
 features: [regexp-named-groups, regexp-match-indices]
 info: |
-  MakeIndicesArray ( S, indices, groupNames )
-    11. For each integer _i_ such that _i_ >= 0 and _i_ < _n_, do
-      e. If _groupNames_ is not *undfined* and _groupNames_[_i_] is not *undefined*, then
-        i. Perform ! CreateDataProperty(_groups_, _groupNames_[_i_], _matchIndicesArray_).
+  MakeIndicesArray ( S, indices, groupNames, hasIndices )
+    13. For each integer _i_ such that _i_ >= 0 and _i_ < _n_, do
+      e. If _i_ > 0 and _groupNames_[_i_ - 1] is not *undefined*, then
+        i. Perform ! CreateDataProperty(_groups_, _groupNames_[_i_ - 1], _matchIndicesArray_).
 ---*/
 
 
 // Properties created on result.groups in textual order.
-let groupNames = Object.getOwnPropertyNames(/(?<fst>.)|(?<snd>.)/u.exec("abcd").indices.groups);
+let groupNames = Object.getOwnPropertyNames(/(?<fst>.)|(?<snd>.)/du.exec("abcd").indices.groups);
 assert.compareArray(groupNames, ["fst", "snd"]);
 
 // // Properties are created with Define, not Set
 // let counter = 0;
 // Object.defineProperty(Object.prototype, 'x', {set() { counter++; }});
```

Comment:
Is this dead code?

Reply:
Not sure why I commented it out originally. It probably should be evaluated.